rpc/sigpipe.go

```diff
@@ -0,0 +1,9 @@
+// +build !darwin
+
+package rpc
+
+import "net"
+
+func DisableSigPipe(c net.Conn) error {
+	return nil
```

Comment:
Although this makes it so that we need to detect whether this is supported on the clients of this library, which makes that code a lot more verbose. With this just doing nothing all that is simpler.